setup.py

```diff
@@ -25,29 +21,6 @@
     from setuptools.command.build_ext import build_ext
     from setuptools.extension import Extension
 
-_HAVE_SPHINX = True
-try:
-    from sphinx.cmd import build as sphinx
-except ImportError:
-    try:
-        import sphinx
-    except ImportError:
-        _HAVE_SPHINX = False
-
-version_ns = {}
-with open("pymongo/_version.py") as fp:
-    exec(fp.read(), version_ns)
-version = version_ns["__version__"]
-
-f = open("README.rst")
-try:
-    try:
-        readme_content = f.read()
-    except BaseException:
-        readme_content = ""
-finally:
-    f.close()
-
 # PYTHON-654 - Clang doesn't support -mno-fused-madd but the pythons Apple
```

Comment:
How does the new build process handle this "-mno-fused-madd" issue? Or is it no longer a concern.

Reply:
Actually, we don't need to use the stdlib `distutils` at all, and the `distutils` that ships with `setuptools` does not add this flag.  I cleaned up this block.